models/src_qualtrics.yml

```diff
@@ -528,7 +528,7 @@ sources:
           - name: unsubscribed
             description: "{{ doc('mailing_list_unsubscribed') }}"
 
-      - name: core_contact
+      - name: core_contact 
```

Comment:
I noticed we are not including source testing on the sources. Usually for models that can be disabled we add the disable config. Is the reason core_contact and core_mailing_list do not have the disable config because there is no source freshness testing?

Reply:
ah that is because i forgot about source freshness! regardless, i'll add the disable config so that folks' DAGs render properly/accurately

but yeah do you have opinions regarding freshness strictness? given this is a survey connector, i could see it being fairly different for different use cases (ie what if you're only sending surveys quarterly?)